RTS_GMLC/timeseries_pointers.json

```diff
@@ -1019,23 +1019,23 @@
         "simulation": "DAY_AHEAD",
         "category": "Area",
         "component_name": "1",
-        "label": "get_max_active_power",
+        "label": "get_peak_active_power",
         "scaling_factor": 2850.0,
         "data_file": "../forecasts/RTS_GMLC_forecasts/load/DAY_AHEAD_regional_Load.csv"
     },
     {
         "simulation": "DAY_AHEAD",
         "category": "Area",
         "component_name": "2",
-        "label": "get_max_active_power",
+        "label": "get_peak_active_power",
         "scaling_factor": 2850.0,
         "data_file": "../forecasts/RTS_GMLC_forecasts/load/DAY_AHEAD_regional_Load.csv"
     },
     {
         "simulation": "DAY_AHEAD",
         "category": "Area",
         "component_name": "3",
-        "label": "get_max_active_power",
+        "label": "get_peak_active_power",
```

Comment:
@claytonpbarrows this is needed in the GMLC repo

Reply:
Are you sure? doing so causes:
```julia
julia> op_problem =
           OperationsProblem(GenericOpProblem, template_uc, sys; optimizer = solver, horizon = 1, balance_slack_variables = true, use_parameters = true, initial_time = DateTime("2020-07-10T00:00:00"))
[ Info: Unit System changed to SYSTEM_BASE
ERROR: ArgumentError: did not find a forecast matching the requested parameters
Stacktrace:
 [1] get_forecast(::Type{InfrastructureSystems.DeterministicInternal}, ::PowerLoad, ::DateTime, ::Millisecond, ::Int64, ::String, ::Int64) at /Users/cbarrows/Documents/repos/InfrastructureSystems.jl/src/component.jl:147
 [2] get_forecast(::Type{Deterministic}, ::PowerLoad, ::DateTime, ::String, ::Int64) at /Users/cbarrows/Documents/repos/InfrastructureSystems.jl/src/component.jl:74
 [3] get_forecast at /Users/cbarrows/Documents/repos/PowerSystems.jl/src/base.jl:955 [inlined]
 [4] get_time_series(::PowerSimulations.PSIContainer, ::PowerLoad, ::String) at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/devices_models/devices/common/get_time_series.jl:15
 [5] _nodal_expression!(::PowerSimulations.PSIContainer, ::InfrastructureSystems.FlattenIteratorWrapper{PowerLoad}, ::Type{DCPPowerModel}, ::Symbol) at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/devices_models/devices/common/nodal_expression.jl:61
 [6] nodal_expression! at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/devices_models/devices/common/nodal_expression.jl:44 [inlined]
 [7] construct_device!(::PowerSimulations.PSIContainer, ::System, ::DeviceModel{PowerLoad,StaticPowerLoad}, ::Type{DCPPowerModel}) at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/devices_models/device_constructors/load_constructor.jl:176
 [8] _build!(::PowerSimulations.PSIContainer, ::OperationsProblemTemplate, ::System) at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/core/operations_problem.jl:407
 [9] build! at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/core/operations_problem.jl:380 [inlined]
 [10] OperationsProblem at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/core/operations_problem.jl:73 [inlined]
 [11] OperationsProblem{GenericOpProblem}(::OperationsProblemTemplate, ::System, ::Nothing; kwargs::Base.Iterators.Pairs{Symbol,Any,NTuple{5,Symbol},NamedTuple{(:optimizer, :horizon, :balance_slack_variables, :use_parameters, :initial_time),Tuple{MathOptInterface.OptimizerWithAttributes,Int64,Bool,Bool,DateTime}}}) at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/core/operations_problem.jl:60
 [12] #OperationsProblem#36 at /Users/cbarrows/Documents/repos/PowerSimulations.jl/src/core/operations_problem.jl:49 [inlined]
 [13] top-level scope at REPL[161]:1
 [14] eval(::Module, ::Any) at ./boot.jl:331
 [15] eval_user_input(::Any, ::REPL.REPLBackend) at /Users/julia/buildbot/worker/package_macos64/build/usr/share/julia/stdlib/v1.4/REPL/src/REPL.jl:86
 [16] run_backend(::REPL.REPLBackend) at /Users/cbarrows/.julia/packages/Revise/BqeJF/src/Revise.jl:1184
 [17] top-level scope at REPL[1]:0
```